protocols/ping/tests/ping.rs

```diff
@@ -221,3 +267,43 @@ impl Arbitrary for MuxerChoice {
         *[MuxerChoice::Mplex, MuxerChoice::Yamux].choose(g).unwrap()
     }
 }
+
+struct DummyBehaviour;
```

Comment:
What is the reason for not using [`libp2p_swarm::DummyBehaviour`](https://docs.rs/libp2p/0.39.1/libp2p/swarm/struct.DummyBehaviour.html)?

Reply:
Didn't know it existed :)

Reply:
On that note, I should continue working on my `libp2p-swarm-test` crate ...